predicates/traffic/traffic.go

```diff
@@ -102,7 +102,7 @@ func (s *spec) Create(args []interface{}) (routing.Predicate, error) {
 
 	p := &predicate{}
 
-	if c, ok := args[0].(float64); ok && 0.0 <= c && c < 1.0 {
+	if c, ok := args[0].(float64); ok && 0.0 <= c {
```

Comment:
We should default the chance to `1.0` in case `c >= 1` to ensure the chance is at the maximum since `rand.Float64() is in the range [0.0, 1.0)` 
https://golang.org/pkg/math/rand/#Float64